common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -86,6 +87,9 @@ object Messages {
                                       device: Uuid,
                                       status: DeviceStatus) extends BusMessage
 
+  final case class CampaignLaunched(namespace: Namespace, updateId: Uuid, devices: Set[Uuid], pkgUri: Uri,
```

Comment:
There is no more json blob for treehub 'packages'. The hash/length from db is enough.